src/panels/config/integrations/integration-panels/zwave/zwave-migration.ts

```diff
@@ -0,0 +1,480 @@
+                            OpenZWave.
+                          </p>
+                          <ol>
+                            <li>Stop the Z-Wave network</li>
+                            <li>
+                              <i
+                                >If running Home Assistant Core in Docker or in
+                                Python VenV:</i
+                              >
+                              Configure and start ozwdaemon
+                            </li>
+                            <li>Set up the OpenZWave integration</li>
+                            <li>
+                              Migrate entities and devices to the new
+                              integration
+                            </li>
+                            <li>Remove legacy Z-Wave integration</li>
+                          </ol>
+                          <p>
+                            <b>
+                              Please take a backup or a snapshot of your
+                              environment before proceeding.
+                            </b>
+                          </p>
+                        </div>
+                        <div class="card-actions">
+                          <mwc-button @click=${this._continue}>
+                            Continue
+                          </mwc-button>
+                        </div>
+                      </ha-card>
+                    `
+                  : ``}
+                ${this._step === 1
+                  ? html`
+                      <ha-card class="content" header="Stop Z-Wave Network">
+                        <div class="card-content">
+                          <p>
+                            We need to stop the Z-Wave network to perform the
+                            migration. Home Assistant will not be able to
+                            control Z-Wave devices while the network is stopped.
+                          </p>
+                          ${this._stoppingNetwork
+                            ? html`
+                                <div class="flex-container">
+                                  <ha-circular-progress
+                                    active
+                                  ></ha-circular-progress>
+                                  <div><p>Stopping Z-Wave Network...</p></div>
+                                </div>
+                              `
+                            : ``}
+                        </div>
+                        <div class="card-actions">
+                          <mwc-button @click=${this._stopNetwork}>
+                            Stop Network
+                          </mwc-button>
+                        </div>
+                      </ha-card>
+                    `
+                  : ``}
+                ${this._step === 2
+                  ? html`
+                      <ha-card class="content" header="Set up ozwdaemon">
```

Comment:
```suggestion
                      <ha-card class="content" header="Set up OZW daemon">
```
We should be consistent in how we use it

Reply:
The name of the daemon is `ozwdaemon`:
https://github.com/OpenZWave/qt-openzwave

Reply:
Then we should change that everywhere else, all other places are using "OZW daemon"